lib/clickhousex/helpers.ex

```diff
@@ -76,7 +76,7 @@ defmodule Clickhousex.Helpers do
       {:ok, ndt} ->
         "'#{NaiveDateTime.to_iso8601(ndt)}'"
 
-      {:error} ->
+      {:error, _reason} ->
```

Comment:
Wow, that wasn't working at all before then

Reply:
Yep, but I think we never really triggered that error in our codebase anyways, so it didn't lead to any visible effect for us :)